Buildings/Experimental/OpenBuildingControl/CDL/Logical/Composite/OnHold.mo

```diff
@@ -0,0 +1,102 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Logical.Composite;
+model OnHold "Block that holds a signal on for a requested time period"
+
+  LessThreshold                                               les1(threshold=
+        holdOnDuration)
+    annotation (Placement(transformation(extent={{-20,-40},{0,-20}})));
+  Continuous.Constant Zero(final k=0)
+    annotation (Placement(transformation(extent={{-100,-40},{-80,-20}})));
+  Timer                                                        timer
+    annotation (Placement(transformation(extent={{20,10},{40,30}})));
+  Modelica.Blocks.Logical.Pre pre
```

Comment:
Since the `CDL.Logical.Pre` block was added to CDL package, it may be better to change the `Modelica.Blocks.Logical.Pre` to `CDL.Logical.Pre`